inc/form_profile.class.php

```diff
@@ -149,6 +149,9 @@ public static function import($forms_id = 0, $form_profile = []) {
       $formFk = PluginFormcreatorForm::getForeignKeyField();
       $form_profile[$formFk] = $forms_id;
 
+      if (!isset($form_profile['_profile']) || $form_profile['_profile'] === null) {
+         return false;
```

Comment:
Should we fail or should we show a warning ?

@orthagh what do you think about this kind of case (will be seen also for all itemtypes in the core